test/print_test_stats.py

```diff
@@ -73,6 +76,51 @@ def parse_reports(folder):
             tests_by_class[class_name].append(test_case)
     return tests_by_class
 
+def build_message(test_suite):
+    return {
+        "normal": {
+            "build_pr": os.environ.get("CIRCLE_PR_NUMBER"),
+            "build_num": os.environ.get("CIRCLE_BUILD_NUM"),
+            "build_sha1": os.environ.get("CIRCLE_SHA1"),
+            "build_branch": os.environ.get("CIRCLE_BRANCH"),
+            "test_name": test_suite.name,
+        },
+        "int": {
+            "test_total_count": len(test_suite.test_cases),
+            "test_total_time": int(test_suite.total_time * 1000),
+            "test_failed_count": test_suite.failed_count,
+            "test_skipped_count": test_suite.skipped_count,
+            "test_errored_count": test_suite.errored_count,
+        },
+    }
+
+def send_report(reports):
+    access_token = os.environ.get("SCRIBE_GRAPHQL_ACCESS_TOKEN")
+    if access_token:
+        print("[DEBUG] scribe access token provided, sending report...")
+        url = "https://graph.facebook.com/scribe_logs"
+        r = requests.post(
+            url,
+            data={
+                "access_token": access_token,
+                "logs": json.dumps(
+                    [
+                        {
+                            "category": "perfpipe_pytorch_test_times",
+                            "message": json.dumps(build_message(reports[name])),
+                            "line_escape": False,
+                        }
+                        for name in sorted(reports.keys())
+                    ]
+                ),
+            },
+        )
+        print(r.text)
+        r.raise_for_status()
+    else:
+        print("[DEBUG] no scribe access token provided, skip sending report!")
+        return
```

Comment:
Let's dismantle pyramid of doom a bit
```suggestion
    if access_token:
        print("[DEBUG] no scribe access token provided, skip sending report!")
        return
    
    print("[DEBUG] scribe access token provided, sending report...")
    url = "https://graph.facebook.com/scribe_logs"
    r = requests.post(
        url,
        data={
            "access_token": access_token,
            "logs": json.dumps(
                [
                    {
                        "category": "perfpipe_pytorch_test_times",
                        "message": json.dumps(build_message(reports[name])),
                        "line_escape": False,
                    }
                    for name in sorted(reports.keys())
                ]
            ),
        },
    )
    print(r.text)
    r.raise_for_status()
```